acdc-core/src/main/resources/reference.conf

```diff
@@ -6,17 +6,17 @@ com.salesforce.mce.acdc {
             // Simple datasource with no connection pooling. The connection pool has already been
             // specified with HikariCP.
             dataSourceClass = "org.postgresql.ds.PGSimpleDataSource"
-            properties = {
-              serverName = "db"
-              serverName = ${?POSTGRES_SERVER}
-              portNumber = "5432"
-              portNumber = ${?POSTGRES_PORT}
-              databaseName = "postgres"
-              databaseName = ${?POSTGRES_DBNAME}
-              user = "postgres"
-              user = ${?POSTGRES_USER}
-              password = ${?POSTGRES_PASSWORD}
-            }
+
+            serverName = "db"
+            serverName = ${?POSTGRES_SERVER}
+            portNumber = "5432"
+            portNumber = ${?POSTGRES_PORT}
+            databaseName = "postgres"
+            databaseName = ${?POSTGRES_DBNAME}
+            user = "postgres"
+            user = ${?POSTGRES_USER}
+            password = ${?POSTGRES_PASSWORD}
+
```

Comment:
can we add a configuration to configure which route to expose metrics, and when that is not configured, disable reporting altogether

Reply:
Yes, that is a good idea.  Simon and I discussed adding configurability to this project.  A configuration endpoint would be useful.